### ISSUE TITLE
Atom won't open path on command line

### ISSUE THREAD

**Issue by trinaryouroboros** [atom/atom#23670]
I was formerly using Fedora 32 and an older version of atom, and when running in konsole terminal:

`atom ./`

It would open atom to the local path and show all files/folders in it.

I upgraded to Fedora 35 recently and am using the latest atom, and this doesn't work anymore.

It just opens to a blank atom with "Add Folders" on the left.

I tried wiping ~/.atom but it had no effect.

**Comment by darangi** [comment]
Does running `atom` only launch the editor?

**Comment by trinaryouroboros** [comment]
Yes, and it only shows whatever path was opened last, not the current directory.

**Comment by jnuernberg** [comment]
Can you try launching it with `atom .` instead of `atom ./`?

**Comment by castillo-marc** [comment]
Experiencing a similar issue on Windows 11 Home using Git Bash as my command line. 

`atom .` only opens up a new window where the Project folder is where atom is installed (C:\Users\marct\AppData\Local\atom\app-1.60.0). 

`atom` opens up the last open Project folder. 

A workaround I've done to open up atom with respect to my command line's current directory is `atom $(pwd)`, which presumably works with any bash CLI, but I'm not sure how to make `atom .`  work again. It used to, and I'm not sure what I did to change that behavior.
